semgrep-core/src/osemgrep/TODO.md

```diff
@@ -0,0 +1,77 @@
+# osemgrep to-do list and notes
+
+The goal of this file is to track progress and share ideas about the
+migration of the Python CLI wrapper to OCaml, known temporarily as
+osemgrep.
+
+## October 10, 2022
+
+by Martin:
+
+- `osemgrep scan` is the only subcommand that was (partially)
```

Comment:
A big TODO is to plug the result returned by semgrep-core and output it in JSON or in text. That's the output/ directory in semgrep-python to port.

Reply:
yes, thanks